### ISSUE TITLE
Required updates to site create procedure

### ISSUE THREAD

**Issue by DarkMatters** [VarynInc/varyn#186]
In order to properly manage and support the Conference milestone we need a have an assigned site admin account for every site. Currently if the Enginesis admin account attempts to manage data objects they are then created/modified/owned by a user not belonging to that site.

In sites table add column site_admin update site table to add the admin, add to siteData (site config) data structure the admin. Update CMS to use site admin as user-id when working site id changes

### SiteCreate procedure

  + Automatically create a site admin $siteName + ‘-admin’ level 85 random initial password
  + Add to CMS users table
  + Associated enginesis admin (9999) to the new site admin in CMS users

- [x] Go through all sites and make sure there is a site admin account
- [x] regenerate site config on live after #189 

### Site Update procedure

  + [x] Allow change of site admin

### Site list procedure

  + [x] Show site admin user-id, user-name

**Comment by DarkMatters** [comment]
Changing site admin is new procedure `SiteAssignAdmin` but I did not add anything to the CMS for this yet. That should be a separate issue.
